great-idea/index.html

```diff
@@ -17,6 +17,76 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
+    <nav>
+      <a href="services.html">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="features.html">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+      
+      <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+    </nav>
+
+    <header>
+      <div class="head-left">
+        <h1>
+          Innovation
+          <br>
+          On
+          <br>
+          Demand</h1>
+
+        <button>Get Started</button>
+      </div>
+    <img src="img/header-img.png" alt="Image of a code snippet.">
+    </header>
+<section class="above-img">
+  <section class = "features">
```

Comment:
need to work on indenting correctly, only need one section and a nested divs here,